Parse/PFAnalytics.m

```diff
@@ -25,9 +25,11 @@ @implementation PFAnalytics
 ///--------------------------------------
 
 + (BFTask *)trackAppOpenedWithLaunchOptions:(NSDictionary *)launchOptions {
-#if TARGET_OS_IPHONE
+#if TARGET_OS_WATCH
+    NSDictionary *userInfo = nil;
+#elif TARGET_OS_IOS
     NSDictionary *userInfo = launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey];
-#else
+#elif TARGET_OS_MAC
```

Comment:
Seeing as how this is set to true for all iOS derivative platforms (e.g. tvOS as well). I wonder if we can come up with a better macro here...